### PR TITLE
Add Microsoft.CodeCoverage dll back

### PR DESCRIPTION
**Pull request by QilongTang**
Please Note:
1. Before submitting the PR, please review [How to Contribute to Dynamo](https://github.com/DynamoDS/Dynamo/blob/master/CONTRIBUTING.md)
2. Dynamo Team will meet 1x a month to review PRs found on Github (Issues will be handled separately)
3. PRs will be reviewed from oldest to newest
4. If a reviewed PR requires changes by the owner, the owner of the PR has 30 days to respond. If the PR has seen no activity by the next session, it will be either closed by the team or depending on its utility will be taken over by someone on the team
5. PRs should use either Dynamo's default PR template or [one of these other template options](https://github.com/DynamoDS/Dynamo/wiki/Choosing-a-Pull-Request-Template) in order to be considered for review.
6. PRs that do not have one of the Dynamo PR templates completely filled out with all declarations satisfied will not be reviewed by the Dynamo team.
7. PRs made to the `DynamoRevit` repo will need to be cherry-picked into all the DynamoRevit Release branches that Dynamo supports. Contributors will be responsible for cherry-picking their reviewed commits to the other branches after a `LGTM` label is added to the PR.

### Purpose

Still confirming but most likely needed for integrator patching so that Microsoft.CodeCoverage is still part of the DynamoCore delivery this year

image

### Declarations

Check these if you believe they are true

- [x] The code base is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions), and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.

### Reviewers

@DynamoDS/dynamo 

### FYIs

(FILL ME IN, Optional) Names of anyone else you wish to be notified of

### CODE REVIEW

File: src/Libraries/DSIronPython/DSIronPython.csproj [hunk 1]
```diff
@@ -1,8 +1,10 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\..\packages\Microsoft.CodeCoverage.16.2.0\build\netstandard1.0\Microsoft.CodeCoverage.props" Condition="Exists('..\..\packages\Microsoft.CodeCoverage.16.2.0\build\netstandard1.0\Microsoft.CodeCoverage.props')" />
```
Comment (mjkkirschner) [inline src/Libraries/DSIronPython/DSIronPython.csproj]: did we import these props and targets previously?

Comment (QilongTang) [reply src/Libraries/DSIronPython/DSIronPython.csproj]: Nope, I can totally remove them if you prefer

Comment (mjkkirschner) [reply src/Libraries/DSIronPython/DSIronPython.csproj]: yeah, lets make minimal changes to the build - we just want this dll to show up but not effect anything else..  - I'm not sure it's possible to add comments to a csproj but if possible that might also be good (IE - TODO remove this!)

Comment (QilongTang) [reply src/Libraries/DSIronPython/DSIronPython.csproj]: @mjkkirschner TODO comments for removal are added below already, I will remove these extra parts

**Comment by QilongTang** [comment]
@DynamoDS/dynamo Please take another look, I verified locally the new dll is included in the build

**Comment by aparajit-pratap** [comment]
LGTM

**Comment by mjkkirschner** [comment]
LGTM

**Pull request merged**